installers/olm/bundle.annotations.yaml

```diff
@@ -26,13 +26,13 @@ annotations:
   # the value from the bundle with the highest semantic version.
   #
   # https://olm.operatorframework.io/docs/best-practices/channel-naming/
-  operators.operatorframework.io.bundle.channels.v1: v2
-  operators.operatorframework.io.bundle.channel.default.v1: v2
+  operators.operatorframework.io.bundle.channels.v1: v5
+  operators.operatorframework.io.bundle.channel.default.v1: v5
```

Comment:
these should be v2

Reply:
https://github.com/percona/percona-postgresql-operator/pull/518/commits/a5fc9520a6dd3b0023091938b40ba45684f77bb6